src/Avalonia.Visuals/Media/FormattedText.cs

```diff
@@ -10,128 +10,109 @@ namespace Avalonia.Media
     /// <summary>
     /// Represents a piece of text with formatting.
     /// </summary>
-    public class FormattedText : AvaloniaDisposable
+    public class FormattedText
```

Comment:
Yeah, the problem is that the impl may still be needed for the next frame so can't be immediately disposed. I considered adding a "dispose when the render has finished" flag but that started looking like manual memory management, which I'd prefer to avoid if possible. We could always add it in future if necessary.